### PR TITLE
Improve `npm test`: lint

### PR DESCRIPTION
**Pull request by Slayer95**

### CODE REVIEW

**Comment by Slayer95** [comment]
Ugh... https://github.com/spalger/gulp-jshint/issues/88

**Comment by Slayer95** [comment]
Hmm, if we are fine with the tag-specific dependency here, I might as well add a sane JSCS reporter.

File: package.json [hunk 1]
```diff
@@ -49,11 +49,12 @@
   ],
   "private": true,
   "devDependencies": {
-    "gulp-util": "~2.2.14",
     "gulp": "~3.8.7",
-    "gulp-jshint": "~1.9.2",
-    "gulp-jscs": "~1.3.1",
+    "gulp-jshint": "git://github.com/spalger/gulp-jshint#c18df3a11",
```
Comment (Zarel) [inline package.json]: um, I'm not sure how I feel about tracking an unmaintained fork...

Comment (Slayer95) [inline package.json]: It's actually just a single commit on top of the main repository, but we can let this stale until this -or a equivalent feature- is released.

Comment (Zarel) [inline package.json]: Are your other changes good, or do they all depend on this?

Comment (Slayer95) [inline package.json]: 39ad898 depends on it... I guess I should take it out and add it later?

Comment (Zarel) [inline package.json]: Feel free to do it however you think would be best.

Comment (Slayer95) [inline package.json]: Ok, did otherwise.

**Pull request merged**